src/test/java/teammates/test/cases/browsertests/StudentProfilePageUiTest.java

```diff
@@ -270,23 +289,31 @@ private void testAjaxPictureUrl() {
 
         ______TS("Typical case: with email and course");
 
-        getProfilePicturePage(instructorId, email, courseId, StudentProfilePicturePage.class).verifyHasPicture();
+        getProfilePicturePage(instructorGoogleId, TestProperties.TEST_INSTRUCTOR_PASSWORD,
+                email, courseId, StudentProfilePicturePage.class).verifyHasPicture();
 
         ______TS("Failure case: instructor does not have privilege");
 
-        getProfilePicturePage(helperId, email, courseId, NotAuthorizedPage.class);
+        String emailStudentForHelperCourse = testData.students.get("studentForHelperCourse").email;
+        String courseIdHelperCourse = testData.students.get("studentForHelperCourse").course;
+
+        emailStudentForHelperCourse = StringHelper.encrypt(emailStudentForHelperCourse);
+        courseIdHelperCourse = StringHelper.encrypt(courseIdHelperCourse);
+        getProfilePicturePage(instructorGoogleId, TestProperties.TEST_INSTRUCTOR_PASSWORD,
+                emailStudentForHelperCourse, courseIdHelperCourse, NotAuthorizedPage.class);
 
         ______TS("Failure case: non-existent student");
 
-        getProfilePicturePage(instructorId, invalidEmail, courseId, EntityNotFoundPage.class);
+        getProfilePicturePage(instructorId, TestProperties.TEST_INSTRUCTOR_PASSWORD,
```

Comment:
The test fails here on a staging server. There is no such Google account `SHomeUiT.instr`.